tsconfig.json

```diff
@@ -17,6 +17,8 @@
     "sourceMap": true,
     "strict": true,
     "target": "esnext",
+    "jsx": "react",
+    "jsxFactory": "h",
```

Comment:
I am concerned we are not using the default here of `React.createElement`.  I can understand why, but it will be potentially surprising to users that we aren't using the default.  At the very least I would like to see updates to the manual along with this PR before we merge it explaining how it is anticipated to be used.

Reply:
I think either is OK. `React.creaetElement` may rather be useful for real use. `react` and `react-dom/server` can be imported in Deno via `dev.jspm.io`.